Gemfile.lock

```diff
@@ -1002,7 +1004,7 @@ DEPENDENCIES
   web-console (= 4.0.4)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.7p221
```

Comment:
Could you stay on the same Ruby version please? 😁

Reply:
2.7.7 is the version given in the ruby version file, but it has been changed in [the previous PR]https://github.com/OpenSourcePolitics/decidim-nyc/pull/46). What version should we keep ?

Reply:
You're right! Then you should switch to 2.7.7p221 good catch! It's all good for me then